pkg/endpoint/bpf.go

```diff
@@ -160,7 +165,11 @@ func (e *Endpoint) writeHeaderfile(prefix string) error {
 		}
```

Comment:
I think the above symlink creation also needs fixing. On 1.8, we could fail to create the header file below, but still create the symlink here (pointing to non-existant file). Then, if user downgrades to 1.7, we will start relying on the old name, so the symlink...

Reply:
Good point, thanks.